src/angular/menu/menu.ts

```diff
@@ -324,7 +324,12 @@ export class SbbMenu implements AfterContentInit, SbbMenuPanel<SbbMenuItem>, OnI
         }
 
         manager.onKeydown(event);
+        return;
     }
+
+    // Don't allow the event to propagate if we've already handled it, or it may
+    // end up reaching other overlays that were opened earlier (see #22694).
```

Comment:
Maybe change as seen below?
```suggestion
    // end up reaching other overlays that were opened earlier (see angular/components#22694).
```